aviasales/src/app/components/button/button.component.sass

```diff
@@ -0,0 +1,14 @@
+button
```

Comment:
Use classes! Don't set styles for tags. It's better, because you can add some library or outer module to your app and there're would be the same tag and styles will be unexpected